GEM-ML/requirements.txt

```diff
@@ -0,0 +1,192 @@
+absl-py==1.4.0
+accumulation-tree==0.6.2
+aenum==3.1.11
+affine==2.4.0
+anyio==3.6.2
+appdirs==1.4.4
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+arrow==1.2.3
+asttokens==2.2.1
+attrs==22.2.0
+backcall==0.2.0
+beautifulsoup4==4.11.1
+bleach==6.0.0
+boto3==1.26.59
+botocore==1.29.59
+bqplot==0.12.36
+branca==0.6.0
+cachetools==5.3.0
+cffi==1.15.1
+charset-normalizer==3.0.1
+click==8.1.3
+click-plugins==1.1.1
+cligj==0.7.2
+colorama==0.4.6
+colour==0.1.5
+comm==0.1.2
+contourpy==1.0.7
+cycler==0.11.0
+dataclasses-json==0.5.7
+debugpy==1.6.6
+decorator==5.1.1
+defusedxml==0.7.1
+dill==0.3.6
+eo-learn-core==1.4.0
+eo-learn-coregistration==1.4.0
+eo-learn-features==1.4.0
+eo-learn-geometry==1.4.0
+eo-learn-io==1.4.0
+eo-learn-mask==1.4.0
+eo-learn-ml-tools==1.4.0
+eo-learn-visualization==1.4.0
+et-xmlfile==1.1.0
+executing==1.2.0
+fastjsonschema==2.16.2
+filelock==3.9.0
+Fiona==1.8.21
+folium==0.14.0
+fonttools==4.38.0
+fqdn==1.5.1
+fs==2.4.16
+fs-s3fs==1.1.1
+GDAL==3.4.3
+gdown==4.6.0
+geojson==3.0.0
+geopandas==0.12.2
+google-auth==2.16.0
+google-auth-oauthlib==0.4.6
+graphviz==0.20.1
+grpcio==1.51.1
+idna==3.4
+imageio==2.25.0
+ipyevents==2.0.1
+ipyfilechooser==0.6.0
+ipykernel==6.20.2
+ipyleaflet==0.17.2
+ipython==8.9.0
+ipython-genutils==0.2.0
+ipytree==0.2.2
+ipywidgets==8.0.4
+isoduration==20.11.0
+jedi==0.18.2
+Jinja2==3.1.2
+jmespath==1.0.1
+joblib==1.2.0
+jsonpointer==2.3
+jsonschema==4.17.3
+jupyter-events==0.6.3
+jupyter_client==8.0.1
+jupyter_core==5.1.5
+jupyter_server==2.1.0
+jupyter_server_terminals==0.4.4
+jupyterlab-pygments==0.2.2
+jupyterlab-widgets==3.0.5
+kiwisolver==1.4.4
+leafmap==0.15.0
+lightgbm==3.3.5
+llvmlite==0.39.1
+Markdown==3.4.1
+MarkupSafe==2.1.2
+marshmallow==3.19.0
+marshmallow-enum==1.5.1
+matplotlib==3.6.3
+matplotlib-inline==0.1.6
+mistune==2.0.4
+munch==2.5.0
+mypy-extensions==0.4.3
+natsort==8.2.0
+nbclassic==0.5.1
+nbclient==0.7.2
+nbconvert==7.2.9
+nbformat==5.7.3
+nest-asyncio==1.5.6
+networkx==3.0
+notebook==6.5.2
+notebook_shim==0.2.2
+numba==0.56.4
+numpy==1.23.5
+oauthlib==3.2.2
+opencv-python-headless==4.7.0.68
+openpyxl==3.0.10
+packaging==23.0
+pandas==1.5.3
+pandocfilters==1.5.0
+parso==0.8.3
+pickleshare==0.7.5
+Pillow==9.4.0
+platformdirs==2.6.2
+prometheus-client==0.16.0
+prompt-toolkit==3.0.36
+protobuf==3.20.3
+psutil==5.9.4
+pure-eval==0.2.2
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycparser==2.21
+pydot==1.4.2
+Pygments==2.14.0
+pyparsing==3.0.9
+pyproj==3.4.1
+pyrsistent==0.19.3
+pyshp==2.3.1
+PySocks==1.7.1
+pystac==1.6.1
+pystac-client==0.6.0
+python-box==6.1.0
+python-dateutil==2.8.2
+python-json-logger==2.0.4
+pytz==2022.7.1
+pyudorandom==1.0.0
+PyWavelets==1.4.1
+pywin32==305
```

Comment:
```
pywin32==305
pywinpty==2.0.10
```

These two can only be installed on Windows, so the `pip install -r requirements.txt` failed for me on MacOS. Please separate the two into `requirements-win.txt` or similar (or remove altogether if not needed).